components/src/status_im/ui/components/react.cljs

```diff
@@ -341,3 +342,86 @@
           keyboard-avoiding-view)
         {:flex 1 :flex-direction :column}
         (apply vector view styles/flex components)]])))
+
+(defn hide-cue [anim-opacity anim-y cue]
```

Comment:
here we have only top-level react components, better to move it here https://github.com/status-im/status-react/tree/develop/src/status_im/ui/components

also could you rename `cue` to something like `cue-atom`

thanks

Reply:
yes.. indeed.. I realized that and I was moving it before I pingd you :)

Reply:
done